packages/rpm/hazelcast.spec

```diff
@@ -37,6 +37,8 @@ rm -rf $RPM_BUILD_ROOT
 echo 'hazelcastDownloadId=CLI_RPM' > "%{buildroot}%{_prefix}/lib/%{name}/%{name}-%{hzversion}/bin/download/hazelcast-download.properties"
 
 %post
+PATH=$PATH:'%{_prefix}/lib/%{name}/%{name}-%{hzversion}/bin'
+export PATH
```

Comment:
This is a bit interesting. By adding `/bin` folder to the path, there will be four commands available to the users: `hz`, `hazelcast`, `hazelcast-start`, and `hazelcast-stop`. All of them have different (some overlapping) purposes. Maybe @Holmistr can comment on that.

Reply:
I know that this might be a bit confusing but we can't do any better. They all serve a different purpose, so we want to keep them there.
But it's definitely a room for improvement in the next releases.